### PR TITLE
DAY 9 TO DAY 11 (WITH MARKDOWN USED)

### PR DESCRIPTION
**Pull request by navjotdadwal**
DAY 9:

QUESTIONS:

HACKEREARTH: TWOSUM
CODECHEF: VILLINE
CODECHEF: MINIAND
######CODECHEF CONTEST(ENIGMA-PLINTH'20 LNMIIT) :######

QUESTIONS:

RECRUIT VILLAGERS(VILLINE)
MINION AND RANGE(MINIAND)
LEARNINGS:

STUDIED CONTAINERS IN C++ STL
DAY 10:

QUESTIONS:

CODEFORCES->BIT++
CODEFORCES->EVEN ODDS
CODEFORCES->DIMA AND FRIENDS
CODEFORCES->JEFF AND DIGITS
DAY 11:

QUESTIONS:

CODEFORCES->FOOTBALL
CODEFORCES->TEAM
CODEFORCES->SALE
CODEFORCES->BICYCLE CHAIN
CODEFORCES->DUBSET
CODEFORCES->K-STRING

### CODE REVIEW

**Comment by navjotdadwal** [comment]
BOSS I HAVE USED MARKDOWN THIS TIME AND UPLOADED ALL THAT WHICH YOU PUT ON HOLD.
SO CAN YOU PLEASE TELL ME THAT I HAVE DONE IT CORRECTLY OR SOME OTHER CHANGES ALSO NEED TO BE DONE.
I HAVE USED MARKDOWN FOM DAY 9 TO DAY 11.

File: navjotdadwal/README.md [hunk 1]
```diff
@@ -203,13 +203,36 @@ QUESTIONS:
...
 
+
+***DAY 9:***
+
+__QUESTIONS:__
+
+
 DAY 9:
 
 QUESTIONS:
```
Comment (rishabhKalakoti) [inline navjotdadwal/README.md]: Remove this part as it is already written in markdown

File: navjotdadwal/README.md [hunk 2]
```diff
@@ -223,15 +246,21 @@ DAY 10:
...
```
Comment (rishabhKalakoti) [inline navjotdadwal/README.md]: Duplicate. Already covered above.

File: navjotdadwal/README.md [hunk 3]
```diff
@@ -223,15 +246,21 @@ DAY 10:
...
 3. CODEFORCES->DIMA AND FRIENDS
 4. CODEFORCES->JEFF AND DIGITS
 
+***DAY 11:***
+
+__QUESTIONS:__
+
 DAY 11:
 
 QUESTIONS:
```
Comment (rishabhKalakoti) [inline navjotdadwal/README.md]: Duplicate.

**Review by rishabhKalakoti — CHANGES_REQUESTED** [review]

**Pull request closed without merging**